src/Akinator.ts

```diff
@@ -95,6 +95,44 @@ export default class Akinator {
     }
   }
 
+  /*
+   * Continue to guess after a "win"(e.g contine guession after a wrong result).
+   */
+  async continue(): Promise<void | Error> {
+    if (!this.uri || !this.urlApiWs) throw new Error(this.noUri);
+    if (!this.uriObj || !this.session || !this.signature) throw new Error(this.noSession);
+
+    const query = new URLSearchParams();
```

Comment:
can we add this to the other functions and maybe put it in a function call?

Reply:
sure